pandas/tests/internals/test_internals.py

```diff
@@ -807,8 +818,15 @@ def test_validate_bool_args(self):
         invalid_values = [1, "True", [1, 2, 3], 5.0]
         bm1 = create_mgr("a,b,c: i8-1; d,e,f: i8-2")
 
+        msg = (
+            r'For argument "inplace" expected type bool, received type int.|'
+            r'For argument "inplace" expected type bool, received type str.|'
+            r'For argument "inplace" expected type bool, received type list.|'
+            r'For argument "inplace" expected type bool, received type float.'
```

Comment:
Maybe leave out the last word to have this less repetitive?

Reply:
At a second look, I think I will just have ```msg``` to be equal to:

```python
msg = "For argument "inplace" expected type bool, received type (int|str|list|float)."
```

Will this be better?

Reply:
I think that should work.  Give it a shot.